### PR TITLE
[DOCFIX] Improve troubleshooting doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
Fixed a few out-of-date places and added some minor improvements

### CODE REVIEW

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -218,12 +227,12 @@ Typical issues:
...
 
 A: Please follow [Running-Alluxio-on-a-Cluster]({{ '/en/deploy/Running-Alluxio-On-a-Cluster.html' | relativize_url }}),
-[Configuring-Alluxio-with-HDFS]({{ '/en/ufs/HDFS.html' | relativize_url }}).
+[Configuring-Alluxio-with-HDFS]({{ '/en/ufs/HDFS.html' | relativize_url }}),
+and [Configuring-Spark-with-Alluxio]({{ '/en/compute/Spark.html' | relativize_url }}).
 
 Tips:
 
 - The best performance gains occur when Alluxio workers are co-located with the nodes of the computation frameworks.
-- You can use Mesos and Yarn integration if you are already using Mesos or Yarn to manage your cluster.
```
Comment (HelloHorizon) [inline docs/en/operation/Troubleshooting.md:226]: Any reason for us to delete this?

File: docs/en/operation/Troubleshooting.md [hunk 2]
```diff
@@ -249,6 +258,8 @@ installation or `spark/conf/` if you customize this file for Spark) has the foll
 </configuration>
 ```
 
+See the doc page for your specific compute framework for detailed setup instructions.
```
Comment (HelloHorizon) [inline docs/en/operation/Troubleshooting.md:261]: (nit)add example link here?

**Review by HelloHorizon — COMMENTED** [review]
@jiacheliu3 thanks for the improvement! I left some minor questions there.

File: docs/en/operation/Troubleshooting.md [hunk 3]
```diff
@@ -376,6 +416,8 @@ A: Alluxio accelerates your system performance by leveraging temporal or spatial
 Alluxio can be configured under a variety of modes, in different production environments.
 Please make sure the Alluxio version being deployed is update-to-date and supported.
 
+## Posting Questions
+
 When posting questions on the [Mailing List](https://groups.google.com/forum/#!forum/alluxio-users)
 or [Slack channel](https://slackin.alluxio.io/), please attach the full environment information, including
```
Comment (apc999) [inline docs/en/operation/Troubleshooting.md:422]: slackin is out of date 
```suggestion
or [Slack channel](https://alluxio.io/slack), please attach the full environment information, including
```

File: docs/en/operation/Troubleshooting.md [hunk 4]
```diff
@@ -384,3 +426,6 @@ or [Slack channel](https://slackin.alluxio.io/), please attach the full environm
 - UnderFileSystem type and version
 - Computing framework type and version
 - Cluster information, e.g. the number of nodes, memory size in each node, intra-datacenter or cross-datacenter
+- Relevant Alluxio configurations like alluxio-site.properties and alluxio-env.sh
```
Comment (apc999) [inline docs/en/operation/Troubleshooting.md:429]: nit
```suggestion
- Relevant Alluxio configurations like `alluxio-site.properties` and `alluxio-env.sh`
```

**Review by apc999 — APPROVED** [review]
Minor comments left

File: docs/en/operation/Troubleshooting.md (hunk 2 above)
Comment (jiacheliu3) [reply docs/en/operation/Troubleshooting.md:261]: I didn't want to enumerate all compute options like spark here and hive here and presto here. We don't have a URL like `https://docs.alluxio.io/os/user/stable/en/compute/` (have to specify spark or hive etc). :(

File: docs/en/operation/Troubleshooting.md (hunk 1 above)
Comment (jiacheliu3) [reply docs/en/operation/Troubleshooting.md:226]: I don't think Mesos or Yarn integration are still there, are they? @apc999

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Comment by HelloHorizon** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Pull request merged**